contrib/builder/rpm/generate.sh

```diff
@@ -57,7 +61,11 @@ for version in "${versions[@]}"; do
 		sqlite-devel # for "sqlite3.h"
 		tar # older versions of dev-tools don't have tar
 	)
-	echo "RUN yum install -y ${packages[*]}" >> "$version/Dockerfile"
+	if [ "$from" == "oraclelinux:7" ]; then
+		echo "RUN yum install --enablerepo=ol7_optional_latest -y ${packages[*]}" >> "$version/Dockerfile"
```

Comment:
There's no command that will enable/disable a repo outside of using the --enablerepo parameter. The only other way I can think of is to add a ${ENABLEREPO} parameter to the yum command and only set it for oraclelinux:7. 

I can build a similar case/esac block for that, so that other vendors could also enable repos if required.